app/src/main/java/com/x8bit/bitwarden/data/platform/datasource/disk/SettingsDiskSourceImpl.kt

```diff
@@ -395,4 +396,15 @@ class SettingsDiskSourceImpl(
         getBoolean(
             key = HAS_USER_LOGGED_IN_OR_CREATED_AN_ACCOUNT_KEY.appendIdentifier(userId),
         ) == true
```

Comment:
The diskSource should not be assuming nullability, it should always be able to return or recieve a null